pkgs/os-specific/linux/trace-cmd/fix-Makefiles.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/Documentation/Makefile b/Documentation/Makefile
+index 21e42fd..b74c912 100644
+--- a/Documentation/Makefile
++++ b/Documentation/Makefile
+@@ -108,7 +108,9 @@ $(IMGS_INSTALL): %.png.install : %.png force
+ 
+ GUI_INSTALL = $(HTML_INSTALL) $(IMGS_INSTALL)
+ 
+-install: $(MAN1_INSTALL) $(MAN5_INSTALL) $(GUI_INSTALL)
++install_man: $(MAN1_INSTALL) $(MAN5_INSTALL)
++
++install: install_man $(GUI_INSTALL)
+ 
+ clean:
+ 	(cd $(obj); \
+diff --git a/Makefile b/Makefile
+index cde45f8..d218ecc 100644
+--- a/Makefile
++++ b/Makefile
+@@ -42,8 +42,8 @@ DESTDIR_SQ = '$(subst ','\'',$(DESTDIR))'
+ 
+ prefix ?= /usr/local
+ bindir_relative = bin
+-bindir = $(prefix)/$(bindir_relative)
+-man_dir = $(prefix)/share/man
++bindir ?= $(prefix)/$(bindir_relative)
```

Comment:
Is this really necessary? The flags passed to `make` take precedence even over `=`.

Reply:
Nice, I didn't know that. I will remove those.